versions.lock

```diff
@@ -12,135 +18,96 @@ com.palantir.safe-logging:preconditions:1.25.0 (3 constraints: 1536d173)
 org.apache.commons:commons-math3:3.2 (1 constraints: 5c0a8ab7)
-org.checkerframework:checker-qual:3.4.0 (2 constraints: 411a465e)
-org.glassfish.hk2:osgi-resource-locator:1.0.3 (2 constraints: 7d23b265)
-org.glassfish.hk2.external:jakarta.inject:2.6.1 (11 constraints: 6ab14a97)
-org.glassfish.jersey.core:jersey-client:2.31 (4 constraints: 78509cb2)
-org.glassfish.jersey.core:jersey-common:2.31 (11 constraints: dfc64234)
-org.glassfish.jersey.core:jersey-server:2.31 (9 constraints: dab3c81c)
-org.glassfish.jersey.media:jersey-media-jaxb:2.31 (1 constraints: d110c2b2)
+org.apache.httpcomponents.client5:httpclient5:5.1.3 (1 constraints: 0b050a36)
+org.apache.httpcomponents.core5:httpcore5:5.1.3 (2 constraints: 9425db57)
+org.apache.httpcomponents.core5:httpcore5-h2:5.1.3 (1 constraints: 3d13093c)
+org.apiguardian:apiguardian-api:1.1.2 (6 constraints: 896455cc)
+org.checkerframework:checker-qual:2.8.1 (1 constraints: 1d0afeb4)
+org.codehaus.mojo:animal-sniffer-annotations:1.17 (1 constraints: ed09d8aa)
+org.glassfish.hk2:hk2-api:2.4.0-b34 (5 constraints: 9d5608c7)
+org.glassfish.hk2:hk2-locator:2.4.0-b34 (4 constraints: 3d490865)
+org.glassfish.hk2:hk2-utils:2.4.0-b34 (2 constraints: 0719352b)
+org.glassfish.hk2:osgi-resource-locator:1.0.1 (2 constraints: 79234465)
+org.glassfish.hk2.external:aopalliance-repackaged:2.4.0-b34 (2 constraints: 0719352b)
+org.glassfish.hk2.external:javax.inject:2.4.0-b34 (6 constraints: 52712a15)
+org.glassfish.jersey.bundles.repackaged:jersey-guava:2.22.2 (1 constraints: 231118d4)
+org.glassfish.jersey.containers:jersey-container-servlet-core:2.22.2 (1 constraints: 3a053b3b)
+org.glassfish.jersey.core:jersey-client:2.22.2 (1 constraints: 3111f1d4)
+org.glassfish.jersey.core:jersey-common:2.22.2 (5 constraints: e964bdb7)
+org.glassfish.jersey.core:jersey-server:2.22.2 (2 constraints: 2b1fcb94)
+org.glassfish.jersey.ext:jersey-entity-filtering:2.22.2 (1 constraints: ee1514e9)
+org.glassfish.jersey.media:jersey-media-jaxb:2.22.2 (1 constraints: 3111f1d4)
+org.glassfish.jersey.media:jersey-media-json-jackson:2.22.2 (1 constraints: 3a053b3b)
 org.immutables:value:2.8.8 (1 constraints: 14051536)
-org.jboss.logging:jboss-logging:3.3.2.Final (2 constraints: 7022d9d6)
-org.jboss.xnio:xnio-api:3.3.8.Final (2 constraints: 6f1a4d45)
-org.jboss.xnio:xnio-nio:3.3.8.Final (1 constraints: c40da530)
+org.javassist:javassist:3.18.1-GA (1 constraints: 570d4740)
+org.jboss.logging:jboss-logging:3.4.1.Final (3 constraints: f03036d7)
+org.jboss.spec.javax.annotation:jboss-annotations-api_1.3_spec:2.0.1.Final (1 constraints: 150fc165)
+org.jboss.spec.javax.servlet:jboss-servlet-api_4.0_spec:2.0.0.Final (1 constraints: 140fba65)
+org.jboss.threads:jboss-threads:3.1.0.Final (2 constraints: 561a9b42)
+org.jboss.xnio:xnio-api:3.8.7.Final (2 constraints: 771a3146)
+org.jboss.xnio:xnio-nio:3.8.7.Final (1 constraints: c80dcb30)
+org.junit.jupiter:junit-jupiter:5.9.0 (1 constraints: 10051f36)
```

Comment:
so this is...odd...it's because these were expressed as test dependencies, so they wind up here. I'm expecting them not to appear in the end state POM though.

Reply:
wc has an internal test module with the same thing. If you’re worried about the locks, opting out of locking configurations in this module will resolve things.